scipy/optimize/tests/test__numdiff.py

```diff
@@ -0,0 +1,275 @@
+from __future__ import division
+
+import numpy as np
+from numpy.testing import (assert_raises, assert_allclose, assert_equal,
+                           assert_, TestCase, run_module_suite, dec,
+                           assert_almost_equal)
+from scipy.optimize import approx_derivatives, check_derivatives
+from scipy.optimize._numdiff import (_adjust_step_1st_order,
+                                     _adjust_step_2nd_order)
+
+
+class TestAdjustStep1stOrder:
```

Comment:
This is an old-style class on python 2, any specific reason to not use a new-style class?